### PR TITLE
[EP-439] QA Phase 2 - Properties to remove

### PR DESCRIPTION
**Pull request by moderateepheezy**
# 📲 What

Remove the following properties from segment.

- session_device_distinct_id
- session_device_manufacturer
- session_device_model
- session_os_version
- session_screen_width
- session_user_agent

# 🤔 Why

Based on feedback from QA on Phase 2 tickets, some Segment event properties aren't needed.

### CODE REVIEW

**Review by tobitech — APPROVED** [review]

File: Library/Tracking/KSRAnalytics.swift [hunk 1]
```diff
@@ -1503,10 +1503,7 @@ public final class KSRAnalytics {
     props["country"] = self.config?.countryCode
     props["display_language"] = AppEnvironment.current.language.rawValue
     props["device_type"] = self.device.deviceType
-    props["device_manufacturer"] = "Apple"
-    props["device_model"] = KSRAnalytics.deviceModel
     props["device_orientation"] = self.deviceOrientation
-    props["device_distinct_id"] = self.distinctId
```
Comment (singhhari) [inline Library/Tracking/KSRAnalytics.swift:1509]: Doesn't look like we are using distinctId for anything else in the file. I would delete this and double check everything is compiling fine.

File: Library/Tracking/KSRAnalytics.swift [hunk 2]
```diff
@@ -1503,10 +1503,7 @@ public final class KSRAnalytics {
     props["country"] = self.config?.countryCode
     props["display_language"] = AppEnvironment.current.language.rawValue
     props["device_type"] = self.device.deviceType
-    props["device_manufacturer"] = "Apple"
-    props["device_model"] = KSRAnalytics.deviceModel
```
Comment (singhhari) [inline Library/Tracking/KSRAnalytics.swift:1507]: Doesn't look like we are using deviceModel for anything else in the file. I would delete this and double check everything is compiling fine.

**Review by singhhari — CHANGES_REQUESTED** [review]

File: Library/Tracking/KSRAnalytics.swift (hunk 1 above)
Comment (moderateepheezy) [reply Library/Tracking/KSRAnalytics.swift:1509]: Thanks, I missed that.

**Review by singhhari — APPROVED** [review]
lgtm

**Pull request merged**